spacy_llm/backends/integration/hf/stablelm.py

```diff
@@ -0,0 +1,126 @@
+from typing import Iterable, Callable, Any, Dict, Optional
+
+from spacy.util import SimpleFrozenList, SimpleFrozenDict
+
+from .base import HuggingFaceBackend
+from ....compat import transformers, torch, has_transformers
+from ....registry.util import registry
+
+if has_transformers:
+
+    class _StopOnTokens(transformers.StoppingCriteria):
+        def __call__(
+            self, input_ids: torch.LongTensor, scores: torch.FloatTensor, **kwargs
+        ) -> bool:
+            stop_ids = [50278, 50279, 50277, 1, 0]
+            for stop_id in stop_ids:
+                if input_ids[0][-1] == stop_id:
+                    return True
+            return False
+
+
+class StableLMHFBackend(HuggingFaceBackend):
```

Comment:
We don't really need the `HF` in there, since we have a `hf` package now. I'd keep a `_HF` postfix in the registry name though, as it's possible that we'll have the same model from different sources (e. g. maybe there'll be a native/curated Dolly model/backend at some point. Not sure whether this will lead to naming conflicts, but I'd rather avoid them). 

Suggestion in https://github.com/explosion/spacy-llm/pull/141/commits/49c27fb18cfa015210bd3c0fabf6b2e05ef5ebe0.